core/src/utilities/helpers/generic-helpers.js

```diff
@@ -285,8 +292,7 @@ class GenericHelpersClass {
   }
 
   /**
-   * Returns a new Object with the same object,
-   * without the keys that were given.
+   * Returns a new Object with the same object,   * without the keys that were given.
```

Comment:
```suggestion
   * Returns a new Object with the same object,   
   * without the keys that were given.
```
Seems like an unnecessary change/typo